benefit/data/sheets/WorkBasketInputFields/WorkBasketInputFields.json

```diff
@@ -10,12 +10,13 @@
   {"LiveFrom": "18/06/2019", "CaseTypeID": "Benefit", "UserRole": "caseworker-sscs-clerk", "CaseFieldID": "directionDueDate", "Label": "Response due date", "DisplayOrder": 9},
   {"LiveFrom": "20/06/2019", "CaseTypeID": "Benefit", "UserRole": "caseworker-sscs-clerk", "CaseFieldID": "createdInGapsFrom", "Label": "Is a new digital case?", "DisplayOrder": 10},
   {"LiveFrom": "20/06/2019", "CaseTypeID": "Benefit", "UserRole": "caseworker-sscs-clerk", "CaseFieldID": "appeal", "ListElementCode": "hearingType", "Label": "Hearing type", "DisplayOrder": 11},
-  {"LiveFrom": "20/06/2019", "CaseTypeID": "Benefit", "UserRole": "caseworker-sscs-clerk", "CaseFieldID": "isScottishCase", "Label": "Is a Scottish case?", "DisplayOrder": 12},
-  {"LiveFrom": "20/06/2019", "CaseTypeID": "Benefit", "UserRole": "caseworker-sscs-clerk", "CaseFieldID": "languagePreferenceWelsh", "Label": "Welsh language flag", "DisplayOrder": 13},
-  {"LiveFrom": "20/06/2019", "CaseTypeID": "Benefit", "UserRole": "caseworker-sscs-clerk", "CaseFieldID": "translationWorkOutstanding", "Label": "Translation outstanding", "DisplayOrder": 14},
-  {"LiveFrom": "20/06/2019", "CaseTypeID": "Benefit", "UserRole": "caseworker-sscs-clerk", "CaseFieldID": "reasonableAdjustmentsOutstanding", "Label": "Alternative letter format required", "DisplayOrder": 15},
-  {"LiveFrom": "20/06/2019", "CaseTypeID": "Benefit", "UserRole": "caseworker-sscs-clerk", "CaseFieldID": "hasUnprocessedAudioVideoEvidence", "Label": "Does case have unprocessed audio/video evidence?", "DisplayOrder": 16},
-  {"LiveFrom": "16/12/2021", "CaseTypeID": "Benefit", "UserRole": "caseworker-sscs-clerk", "CaseFieldID": "dwpDueDate", "Label": "FTA response due date", "DisplayOrder": 17},
+  {"LiveFrom": "17/08/2022", "CaseTypeID": "Benefit", "UserRole": "caseworker-sscs-clerk", "CaseFieldID": "isConfidentialCase", "Label": "Is a Confidential Case?", "DisplayOrder": 12},
+  {"LiveFrom": "20/06/2019", "CaseTypeID": "Benefit", "UserRole": "caseworker-sscs-clerk", "CaseFieldID": "isScottishCase", "Label": "Is a Scottish case?", "DisplayOrder": 13},
+  {"LiveFrom": "20/06/2019", "CaseTypeID": "Benefit", "UserRole": "caseworker-sscs-clerk", "CaseFieldID": "languagePreferenceWelsh", "Label": "Welsh language flag", "DisplayOrder": 14},
+  {"LiveFrom": "20/06/2019", "CaseTypeID": "Benefit", "UserRole": "caseworker-sscs-clerk", "CaseFieldID": "translationWorkOutstanding", "Label": "Translation outstanding", "DisplayOrder": 15},
+  {"LiveFrom": "20/06/2019", "CaseTypeID": "Benefit", "UserRole": "caseworker-sscs-clerk", "CaseFieldID": "reasonableAdjustmentsOutstanding", "Label": "Alternative letter format required", "DisplayOrder": 16},
+  {"LiveFrom": "20/06/2019", "CaseTypeID": "Benefit", "UserRole": "caseworker-sscs-clerk", "CaseFieldID": "hasUnprocessedAudioVideoEvidence", "Label": "Does case have unprocessed audio/video evidence?", "DisplayOrder": 17},
```

Comment:
Don't forget to modify the ordering of the nonprod sheets too for these

Reply:
This field is only added in prod one and not in non-prod so ordering in non-prod will remain same.

Reply:
It does, you need to change the DisplayOrder on the nonprod sheets too. For example, for caseworker-sscs-superuser, dwpDueDate is now the same order as dwpIsOfficerAttending in the nonprod sheet. 
You need to increment the DisplayOrder on the non prod sheets too

Reply:
@TeaWithLucas  does that mean I also need to add that new field on non-prod as well?

Reply:
> It does, you need to change the DisplayOrder on the nonprod sheets too. For example, for caseworker-sscs-superuser, dwpDueDate is now the same order as dwpIsOfficerAttending in the nonprod sheet. You need to increment the DisplayOrder on the non prod sheets too

updated non-prod ordering as well

Reply:
> @TeaWithLucas does that mean I also need to add that new field on non-prod as well?

No, prod fields are added combined with nonprod sheets outside of production

Reply:
> > It does, you need to change the DisplayOrder on the nonprod sheets too. For example, for caseworker-sscs-superuser, dwpDueDate is now the same order as dwpIsOfficerAttending in the nonprod sheet. You need to increment the DisplayOrder on the non prod sheets too
> 
> updated non-prod ordering as well

Can you update WorkBasketInputFields-postponements-nonprod.json too please

Reply:
updated this as well